core/domain/prod_validators_test.py

```diff
@@ -9380,6 +9380,161 @@ def test_object_validation_failure(self):
                 expected_output, sort=True, literal_eval=False)
 
 
+class ReviewerAndSuggestionCountsModelValidatorTests(
+        test_utils.AuditJobsTestBase):
+
+    translation_reviewer_counts_by_lang = {
+        'hi': 0,
+        'en': 1
+    }
+
+    translation_suggestion_counts_by_lang = {
+        'fr': 6,
+        'en': 5
+    }
+
+    question_reviewer_count = 1
+    question_suggestion_count = 4
+
+    negative_count = -1
+    sample_language_code = 'en'
+
+    def setUp(self):
+        super(ReviewerAndSuggestionCountsModelValidatorTests, self).setUp()
+
+        self.job_class = (
+            prod_validation_jobs_one_off
+            .ReviewerAndSuggestionCountsModelAuditOneOffJob
+        )
+
+    def test_model_validation_success_when_model_has_non_zero_counts(self):
+        suggestion_models.ReviewerAndSuggestionCountsModel(
+            id=suggestion_models.REVIEWER_AND_SUGGESTION_COUNTS_MODEL_ID,
+            translation_reviewer_counts_by_lang=(
+                self.translation_reviewer_counts_by_lang),
+            translation_suggestion_counts_by_lang=(
+                self.translation_suggestion_counts_by_lang),
+            question_reviewer_count=self.question_reviewer_count,
+            question_suggestion_count=self.question_suggestion_count
+        ).put()
+        expected_output = [(
+            u'[u\'fully-validated ReviewerAndSuggestionCountsModel\', 1]')]
+
+        self.run_job_and_check_output(
+            expected_output, sort=False, literal_eval=False)
+
+    def test_model_validation_success_when_model_has_default_values(self):
```

Comment:
Suggest also having a test that runs the validation job in the initial state (i.e. no model created).